docs/changes/16181.other.rst

```diff
@@ -0,0 +1,3 @@
+Change the default value of ``copy`` arguments in public APIs from ``False`` to
+``None`` if Numpy 2.0 or newer is installed.
+For details, see :ref:`What's New in Astropy 6.1?<_whatsnew-6.1-copy-semantics>`_
```

Comment:
I think we generally try to avoid putting links in changelog fragments (because they are more likely to get out of date), but I'm not sure whether this one is OK - let's ping @pllim!

Reply:
Yeah:

1. That ref is wrong. Should have used `whatsnew-6.1-copy-semantics`.
2. I also remember some past pain where we linked to something in What's New but then the link broke after release. But I don't remember exactly what broke. @saimn , do you?

Reply:
p.s. Maybe it is not the change log fragment but the what's new. See https://github.com/astropy/astropy/pull/16181#issuecomment-2000243710

Reply:
Ah, that would do it.

Reply:
thank you ! should be fixed now !